README.md

```diff
@@ -93,6 +93,21 @@ This site exposes a public JSON API powered by [Next.js API routes](https://next
 - [`/api/r/:emoji`](https://scrapbook.hackclub.com/api/r/hardware) – Get all posts tagged with a specific emoji
 - [`/:username.png`](https://scrapbook.hackclub.com/sampoder.png) – Get a user's avatar as an image URL
 
+## Running Locally
+
+1. Clone this repository
+   - `git clone git@github.com:hackclub/scrapbook.git`
```

Comment:
```suggestion
   - `git clone git@github.com:hackclub/scrapbook.git; cd scrapbook`
```

Reply:
Good call! Fixed in [c21bab9](https://github.com/hackclub/scrapbook/pull/280/commits/c21bab97ca9f15305be40debb8d45fdcf543c85b)